scripts/install.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+
+{ # this ensures the entire script is downloaded #
+
+BINARY_DEST=${2:-/usr/local/bin}
+BINARY_NAME="rhoas"
+SRC_ORG="bf3fc6c"
+SRC_REPO="cli"
+OS_TYPE="linux"
+OS_LONG_BIT=""
+API_BASE_URL="https://api.github.com"
+API_RELEASE_URL="${API_BASE_URL}/repos/${SRC_ORG}/${SRC_REPO}/releases/latest"
+DOWNLOAD_DIR="/tmp"
+
+has_in_path() {
+  type "$1" > /dev/null 2>&1
+}
+
+source_file() {
+  # shellcheck disable=SC1090
+  source "$1" 2> /dev/null
+}
+
+# get OS type
+if [[ "$OSTYPE" == "darwin"* ]]; then
+  OS_TYPE="macOS"
+fi
+
+MACHINE_TYPE=$(uname -m)
+if [ "${MACHINE_TYPE}" == 'x86_64' ]; then
+  OS_LONG_BIT="64"
+else
+  OS_LONG_BIT="386"
+fi
+
+if ! has_in_path "curl"; then
+  echo "curl is required to download this binary"
+  exit 1
+fi
+
+LATEST_TAG=$(curl -s "${API_RELEASE_URL}" \
+| grep "tag_name.*" \
+| cut -d '"' -f 4)
+
+ASSET_NAME="${BINARY_NAME}_${LATEST_TAG}_${OS_TYPE}_amd${OS_LONG_BIT}"
+ASSET_NAME_COMPRESSED="${ASSET_NAME}.tar.gz"
+
+DOWNLOAD_URL=$(curl -s "${API_RELEASE_URL}" \
+| grep "browser.download_url.*${ASSET_NAME_COMPRESSED}" \
+| cut -d '"' -f 4)
+
+cd "$DOWNLOAD_DIR" || exit
+
+# wget is faster, use it to download the release if available
+if has_in_path "wget"; then
+  wget "$DOWNLOAD_URL"
```

Comment:
I would suppres excessive logs by piping those to null:

```
Password:
--2021-02-19 10:41:00--  https://github.com/bf3fc6c/cli/releases/download/0.16.0/rhoas_0.16.0_macOS_amd64.tar.gz
Resolving github.com (github.com)... 140.82.121.3
Connecting to github.com (github.com)|140.82.121.3|:443... connected.
HTTP request sent, awaiting response... 302 Found
Location: https://github-releases.githubusercontent.com/339445467/3f14b000-7143-11eb-8350-f4719674a261?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAIWNJYAX4CSVEH53A%2F20210219%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20210219T104101Z&X-Amz-Expires=300&X-Amz-Signature=a1ee7ecad0d30de172b8135453152f1aca62f6c4b046a24d9865e3a509157f78&X-Amz-SignedHeaders=host&actor_id=0&key_id=0&repo_id=339445467&response-content-disposition=attachment%3B%20filename%3Drhoas_0.16.0_macOS_amd64.tar.gz&response-content-type=application%2Foctet-stream [following]
--2021-02-19 10:41:01--  https://github-releases.githubusercontent.com/339445467/3f14b000-7143-11eb-8350-f4719674a261?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAIWNJYAX4CSVEH53A%2F20210219%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20210219T104101Z&X-Amz-Expires=300&X-Amz-Signature=a1ee7ecad0d30de172b8135453152f1aca62f6c4b046a24d9865e3a509157f78&X-Amz-SignedHeaders=host&actor_id=0&key_id=0&repo_id=339445467&response-content-disposition=attachment%3B%20filename%3Drhoas_0.16.0_macOS_amd64.tar.gz&response-content-type=application%2Foctet-stream
Resolving github-releases.githubusercontent.com (github-releases.githubusercontent.com)... 185.199.109.154, 185.199.111.154, 185.199.110.154, ...
Connecting to github-releases.githubusercontent.com (github-releases.githubusercontent.com)|185.199.109.154|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 10715145 (10M) [application/octet-stream]
Saving to: ‘rhoas_0.16.0_macOS_amd64.tar.gz’
```

This is poluting a lot and do not allow us to provide nice information